nxrm-ha/nx-values.yaml

```diff
@@ -0,0 +1,71 @@
+# Declare variables to be passed into your templates.
+namespaces:
+  nexusNs:
+    enabled: false
+    name: "nexusrepo"
+statefulset:
+  replicaCount: 2
+  additionalVolumes:
+    - name: nfs-blob
+      persistentVolumeClaim:
+        claimName: nfs-blob-pvc
+  additionalVolumeMounts:
+    - name: nfs-blob
+      mountPath: "/nfs-blob"
+  container:
+    resources:
+      # See help documentation, these are minimum system requirements
+      requests:
+        cpu: 1
+        memory: "2Gi"
+      limits:
+        cpu: 1
+        memory: "2Gi"
+    env:
+      nexusDBName: kind_nxha77db
+      nexusDBPort: 5432
+  livenessProbe:
+    failureThreshold: 6
+    initialDelaySeconds: 20
+    path: /
+    periodSeconds: 60
+  readinessProbe:
+    failureThreshold: 6
+    initialDelaySeconds: 20
+    path: /
+    periodSeconds: 60
+storageClass:
+  enabled: false # For built-in storage class, set storageClass.enabled to false
+  name: nfs 
+pvc:
+  accessModes: ReadWriteOnce
+  storage: 1Gi
+  volumeClaimTemplate:
+    enabled: true
+secret:
+  dbSecret:
+    enabled: true
+  db:
+    user: nexus
+    password: nexus123
+    host: 172.17.0.1
+  nexusAdminSecret:
+    enabled: true # Enable to apply nexus-admin-secret.yaml which allows you to the initial admin password for nexus repository
+    adminPassword: admin123 #You should change this when you login for the first time
```

Comment:
I worry that people using the helm chart may not change the admin password, we don't want to encourage the use of admin123 if we can avoid it. Unfortunately, I think for helm charts they may not be able to see the contents of a randomly generated password file. If customers have the ability to override this setting it may be better not to set enabled and adminPassword at all and let customers set it as part of the install process.